lib_comfyui/default_workflow_types.py

```diff
@@ -20,27 +20,43 @@
     default_workflow=external_code.AUTO_WORKFLOW,
     types='LATENT',
 )
+postprocess_latent_workflow_type = external_code.WorkflowType(
+    base_id='postprocess_latent',
+    display_name='Postprocess (latent)',
+    default_workflow=external_code.AUTO_WORKFLOW,
+    types='LATENT',
+)
 postprocess_workflow_type = external_code.WorkflowType(
     base_id='postprocess',
     display_name='Postprocess',
     default_workflow=external_code.AUTO_WORKFLOW,
     types='IMAGE',
 )
-postprocess_latent_workflow_type = external_code.WorkflowType(
-    base_id='postprocess_latent',
-    display_name='Postprocess (latent)',
+postprocess_image_workflow_type = external_code.WorkflowType(
+    base_id='postprocess_image',
+    display_name='Postprocess image',
     default_workflow=external_code.AUTO_WORKFLOW,
-    types='LATENT',
+    types='IMAGE',
+    max_amount_of_io_nodes=[None, 1],
```

Comment:
have you considered splitting this into two `Optional[int]` fields instead of just one?